_maps/map_files/Deltastation/DeltaStation2.dmm

```diff
@@ -7188,8 +7203,9 @@
 /obj/machinery/atmospherics/pipe/heat_exchanging/simple{
 	dir = 4
 	},
+/obj/machinery/atmospherics/pipe/heat_exchanging/simple,
 /obj/structure/lattice/catwalk,
-/turf/open/space,
+/turf/open/space/basic,
```

Comment:
There a reason these are switched to /basic?

Reply:
Nope, didn't actually know I did that. Might've happened when I was trying to remove the d1 and d2 variables or due to the most recent update with the white ship.